gluten-ut/src/test/scala/org/apache/spark/sql/GlutenSQLQueryTestSuite.scala

```diff
@@ -209,7 +209,26 @@ class GlutenSQLQueryTestSuite extends QueryTest with SharedSparkSession with SQL
   /** List of supported cases to run, in lower case.
    * Please add to the supported list after enabling a sql test.
    */
-  private val supportedList: Set[String] = Set()
+  private val supportedList: Set[String] = Set(
```

Comment:
Please do a rebase for the supported list, thanks.

Reply:
Rebased.